pkg/agent/route/route_linux.go

```diff
@@ -642,6 +642,32 @@ func (c *Client) restoreIptablesData(podCIDR *net.IPNet, podIPSet, localAntreaFl
 		}...)
 	}
 
+	// This generates the rule to masquerade the packets destined for hostPort which backend is AntreaIPAM VLAN Pod.
+	// For simplicity, in the following:
+	//   - per-Node IPAM Pod is referred to as Pod.
+	//   - Antrea IPAM Pod without VLAN is referred to as Antrea Pod.
```

Comment:
Antrea Pod -> AntreaIPAM Pod (otherwise hard to understand)

Reply:
Refactored.